File: src/C#/CreationalPatterns/AbstractFactory/Furniture/Products/ScandinavianDiningTable.cs
using Furniture.Interfaces;
using System;
using System.Collections.Generic;
using System.Text;

namespace Furniture.Products
{
    public class ScandinavianDiningTable : IFurniture
    {
        public void ShowFurnitureStyle()
        {
            Console.WriteLine("I am a Scandinavian dining table");
        }
    }
}

File: src/PHP/CreationalPatterns/AbstractFactory/Furniture/Products/ScandinavianDiningTable.php
<?php

namespace CreationalPatterns\AbstractFactory\Furniture\Products;

use CreationalPatterns\AbstractFactory\Furniture\Interfaces\IFurniture;

class ScandinavianDiningTable implements IFurniture {
    public function showFurnitureStyle(): void
    {
        echo "I am a Scandinavian dining table\n";
    }
}